src/types/player.types.ts

```diff
@@ -97,3 +97,36 @@ export interface ResumePlaybackParams {
   position_ms?: number;
   offset?: PositionOffset | URIOffset;
 }
+
+export interface GetRecentlyPlayedTracksOptionalParams {
+  limit?: number;
+  after?: number;
+  before?: number;
+}
+
+interface PlaybackCursorParams {
```

Comment:
Change the name of the interface to CurosorProps and move the interface to shared.types.ts and import from that file to this file